crypto_box/src/lib.rs

```diff
@@ -94,6 +94,58 @@
 //! assert_eq!(&plaintext[..], &decrypted_plaintext[..]);
 //! ```
 //!
+//! ## Customized Associated Data Usage
```

Comment:
Maybe this section should be titled something like "Choosing XChaCha20Poly1305 vs XSalsa20Poly1305"